roles/default/kiali-deploy/defaults/main.yml

```diff
@@ -188,6 +188,24 @@ kiali_defaults:
     web_root: ""
     web_schema: ""
 
+  health_config:
```

Comment:
Please put all settings in alphabetical sort order. So this "health_config" should go up after the `external_services` section and before the `identity` section ([see line 148 here](https://github.com/kiali/kiali-operator/blob/f8eb903fa270352bdba671d711419bee4e98bff6/roles/default/kiali-deploy/defaults/main.yml#L148))